### PR TITLE
Discussion - add specific error message

### PR DESCRIPTION
**Pull request by mchv**
## What does this change?

Between the time a user start replying to a comment and the comment is posted the parent comment may has been moderated or deleted. In that case the current user experience display a generic error message:

ce33e9ae-398f-47d7-9154-ce5004fa2be7

See discussion API code:

[Comment deleted](https://github.com/guardian/discussion-api/blob/master/discussion-api/src/main/scala/com.gu.discussion.api/api/DiscussionApi.scala#L111-L113)

``` scala
val parentCommentOption = replyToCommentId map { parentId ⇒
      repositories.writeableComment(parentId) getOrElse NotFoundException(ApiErrorMessages.commentNotFound)
    }
```

[Comment moderated](https://github.com/guardian/discussion-api/blob/master/discussion-api/src/main/scala/com.gu.discussion.api/api/DiscussionApi.scala#L137-L139)

``` scala
if (Switchboard.noReplyModComments.enabled && (parent.status != "visible")) {
          InvalidApiRequest(ApiErrorMessages.commentParentModerated)
        }
```

### CODE REVIEW

**Comment by jamesgorrie** [comment]
👍 Niche.

File: static/src/javascripts/projects/common/modules/discussion/comment-box.js [hunk 1]
```diff
@@ -72,6 +72,7 @@ CommentBox.prototype.errorMessages = {
     COMMENT_TOO_LONG: 'Your comment must be fewer than 5000 characters long.',
     USER_BANNED: 'Commenting has been disabled for this account (<a href="/community-faqs#321a">why?</a>).',
     DISCUSSION_CLOSED: 'Sorry your comment can not be published as the discussion is now closed for comments.',
+    PARENT_COMMENT_MODERATED: 'Sorry the comment can not be published as the comment you reply to as been moderated since',
```
Comment (gtrufitt) [inline static/src/javascripts/projects/common/modules/discussion/comment-box.js]: I think you mean 'comment you replied to' :)

Comment (mchv) [inline static/src/javascripts/projects/common/modules/discussion/comment-box.js]: @gtrufitt thanks updated!

File: static/src/javascripts/projects/common/modules/discussion/comment-box.js [hunk 2]
```diff
@@ -72,6 +72,7 @@ CommentBox.prototype.errorMessages = {
     COMMENT_TOO_LONG: 'Your comment must be fewer than 5000 characters long.',
     USER_BANNED: 'Commenting has been disabled for this account (<a href="/community-faqs#321a">why?</a>).',
     DISCUSSION_CLOSED: 'Sorry your comment can not be published as the discussion is now closed for comments.',
+    PARENT_COMMENT_MODERATED: 'Sorry the comment can not be published as the comment you replied to as been moderated since.',
```
Comment (JonNorman) [inline static/src/javascripts/projects/common/modules/discussion/comment-box.js]: does it also need to be "has been moderated"*

Comment (mchv) [inline static/src/javascripts/projects/common/modules/discussion/comment-box.js]: @JonNorman you are right. fixing now

**Pull request merged**